pkgs/applications/science/biology/bedops/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, zlib, bzip2, jansson, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bedops";
+  version = "2.4.39";
+
+  src = fetchFromGitHub {
+    owner = "bedops";
+    repo = "bedops";
+    rev = "v${version}";
+    sha256 = "sha256-vPrut3uhZK1Eg9vPcyxVNWW4zKeypdsb28oM1xbbpJo=";
+  };
+
+  buildInputs = [ zlib bzip2 jansson ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  preConfigure = ''
+    # We use nixpkgs versions of these libraries
+    rm -rf third-party
+    sed -i '/^LIBS/d' system.mk/*
```

Comment:
Please use substituteInPlace where possible because it fails if something can't be found unlike sed.

Reply:
I've tried to address this and have replaced sed where I could with substituteInPlace. There's a couple of places though where I need to operate on whole lines based on a partial regex match, which I don't think can be done with substituteInPlace. Is it ok to leave these to sed?

Reply:
Yeah thats okay.